crates/ilp-node/tests/redis/payments_incoming.rs

```diff
@@ -137,11 +132,8 @@ async fn payments_incoming() {
     }
 
```

Comment:
Just realized as we've been discussing this internally at least related to #663. The tests should have a sleep in the beginning similar to https://github.com/interledger-rs/interledger-rs/blob/98eca405495aa3b78c6eb45fd4d4dd31f9e2d1e3/crates/ilp-node/tests/redis/exchange_rates.rs#L34-L35

Otherwise as we've seen with #663 there will be there could be the rare failure as one of the nodes isn't yet ready to serve the ws connection, or the nodes haven't yet exchanged routing information.

```suggestion
    // FIXME: similar to exchange_rates::coincap and others; need to await for the nodes to be ready.
    tokio::time::delay_for(Duration::from_millis(1000)).await;
```